core/settings.py

```diff
@@ -36,7 +37,14 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = env("DEBUG")
 
-ALLOWED_HOSTS = []
+DEBUG_TOOLBAR_ENABLED = DEBUG and "test" not in sys.argv
+
+ALLOWED_HOSTS = ["*"]
```

Comment:
I recommend managing allowed_hosts explicitly from day 1. This reduces headache and troubleshooting when it's time to lockdown and deploy. Right now it's localhost and 127.x.x. later you might use an environment variable to set ALLOWED_HOSTS appropriately.
 `ALLOWED_HOSTS = env.list("ALLOWED_HOSTS", default=["localhost"])`